lib/eventsource-polyfill.js

```diff
@@ -1,5 +1,7 @@
 var EventSource = require('./eventsource')
 if (typeof window === 'object') {
-  window.EventSource = window.EventSource || EventSource
+  if (!window.EventSource) window.EventSource = EventSource
+  module.exports = window.EventSource
```

Comment:
Don't bother checking whether `module` is undefined, as `./eventsource` wouldn't work were that the case anyway.